pyvrp/plotting/plot_objectives.py

```diff
@@ -38,17 +38,12 @@ def _plot(x, y, *args, **kwargs):
         ax.plot(x[num_to_skip:], y[num_to_skip:], *args, **kwargs)
 
     x = 1 + np.arange(result.num_iterations)
-    y = [d.best_cost for d in result.stats.infeas_stats]
-    _plot(x, y, label="Infeas. best", c="tab:red")
 
-    y = [d.avg_cost for d in result.stats.infeas_stats]
-    _plot(x, y, label="Infeas. avg.", c="tab:red", alpha=0.3, ls="--")
+    y = [d.current_cost for d in result.stats.data]
```

Comment:
Not plotting candidate because it's a lot of visual noise:
![image](https://github.com/user-attachments/assets/ce7ea50a-11b3-45d4-ba88-7b0540225790)